include/conf/load.hpp

```diff
@@ -55,6 +89,25 @@ namespace color_coded
       while(std::getline(ifs, tmp))
       { args.emplace_back(detail::make_absolute(std::move(tmp), base)); }
 
+      return args;
+    }
+
+    inline args_t load(std::string const &file, std::string const &filetype, std::string const &fileName = "")
```

Comment:
Please don't use default arguments; they're a maintanence nightmare. We can just overload `load` like:

```cpp
inline args_t load(std::string const &file, std::string const &filetype)
{ return load(file, filetype, ""); }
```